examples/image-selector/main.tf

```diff
@@ -0,0 +1,8 @@
+module "image_lookup" {
+  providers = {
+    ibm = ibm
+  }
```

Comment:
why are you adding this? This is not needed? It will automatically use the provider thats defined.